docker-compose.yml

```diff
@@ -5,13 +5,13 @@ x-default-variables: &default-variables
   LOG_FORMAT: ${LOG_FORMAT-json}
   MANIFEST_PATH: ${MANIFEST_PATH-/manifests}
   TEST_DATA: ${TEST_DATA-}
-  HTTP_PORT: ${HTTP_PORT-}
+  HTTP_PORT: ${HTTP_PORT-8080}
   HTTP_HOST: ${HTTP_HOST-0.0.0.0}
-  HEALTH_PORT: ${HEALTH_PORT-}
-  DB_TLS_SSLMODE: ${DB_TLS_SSLMODE-}
-  DB_TLS_CERT: ${DB_TLS_CERT-}
-  DB_TLS_KEY: ${DB_TLS_KEY-}
-  DB_TLS_CA: ${DB_TLS_CA-}
+  HEALTH_PORT: ${HEALTH_PORT-8081}
+  DB_TLS_SSLMODE: ${DB_TLS_SSLMODE-verify-ca}
```

Comment:
As we did in Orchestrate, don't set values in docker compose but in .env so we can make custome settings with it, otherwise for circleCI I believe it would be hard to know what is default and by is overwritten etc